src/explorer/trialApp/TrialAppClient.ts

```diff
@@ -51,17 +53,50 @@ export class TrialAppClient implements IFilesClient {
         return `https://${this.metadata.scmHostName}`;
     }
 
-    public get defaultHostName(): string {
-        return this.metadata.hostName;
-    }
-
     public get defaultHostUrl(): string {
-        return `https://${this.metadata.hostName}`;
+        return `https://${this.fullName}`;
```

Comment:
Usually "fullName" looks like `emjnode1`, "hostName" looks like `emjnode1.azurewebsites.net`, and "hostUrl" looks like `https://emjnode1.azurewebsites.net`. Something seems amiss with the naming here.

Does `IAppSettingsClient` even need `fullName`?

Reply:
> Does `IAppSettingsClient` even need `fullName`?
If you're asking if it's used, then yes. But if you're asking if it could be removed from `IAppSettingsClient` and refactor the code that references `client.fullName` be refactored then it could probably be removed.

I am fixing those properties. There is actually a url property in the metadata I should have been using. And a siteName prop for fullName.